tools/gen-template-doc/gen_template_docs.py

```diff
@@ -71,7 +70,9 @@
          "rhpam74-kieserver-openshift:1.0": "../../../kieserver/image.yaml[`rhpam-7/rhpam74-kieserver-openshift`]",
          "rhpam75-kieserver-openshift:1.0": "../../../kieserver/image.yaml[`rhpam-7/rhpam75-kieserver-openshift`]",
          "jboss-processserver64-openshift:1.4": "../../image.yaml[`jboss-processserver64-openshift`]",
-         "jboss-decisionserver64-openshift:1.4": "../..iamge.yaml[`jboss-decisionserver64-openshift`]"}
```

Comment:
@spolti Did we skip 1.5 here for decisionserver and processserver? The images streams do include 1.4, 1.5, and 1.6...